eo-maven-plugin/src/main/java/org/eolang/maven/ResolveMojo.java

```diff
@@ -80,6 +82,14 @@ public final class ResolveMojo extends SafeMojo {
     @Parameter(property = "eo.discoverSelf", required = true, defaultValue = "false")
     private boolean discoverSelf;
 
+    /**
+     * Fail resolution process on conflicting dependencies.
+     * @since 1.0
+     * @checkstyle MemberNameCheck (7 lines)
+     */
+    @Parameter(property = "eo.ignVerConflicts", required = true, defaultValue = "false")
+    private boolean ignVerConflicts;
```

Comment:
@mximp let's name it `ignoreVersionConflicts`, it sounds more natural

Reply:
Corrected.